### ISSUE TITLE
print renderer in the boilerplate

### ISSUE THREAD

**Issue by umlaeute** [SoundScapeRenderer/ssr#323]
as a very casual user of `ssr`, i find it a bit confusing that it aborts on my pipewire-enabled system, but `ssr-vbap` does not.

the underlying reason is of course that
- `ssr` by calls `ssr-binaural` by default
- `ssr-binaural` requires HRIR files in the correct samplerate
- the HRIR-files that ship with `ssr` are 44.1kHz
- pipewire uses 48kHz as the samplerate (#32, #68)

in any case, what is confusing me is that there's no information about the actual renderer when `ssr` starts up.
instead i'm only greeted by big (and nice but generic) boilerplate:

```sh
$ ssr-vbap

       ___     
      /  ___   
  ___/  /  ___ 
    ___/  /    SSR (SoundScape Renderer) 0.5.0~dfsg
         /     
               
Website: <http://spatialaudio.net/ssr/>
Contact: <ssr@spatialaudio.net>

Copyright © 2016 Division of Applied Acoustics, Chalmers University of Technology
Copyright © 2014 Institut für Nachrichtentechnik, Universität Rostock
Copyright © 2012 Quality & Usability Lab, Telekom Innovation Laboratories, TU Berlin

License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
```

no mention of anything VBAP (or whatever renderer is chosen).

this is probably less a problem when i explicitly select the renderer (either with `ssr-binaural` or `ssr --binaural`), but with the default renderer it creates extra cognitive load.

**Comment by umlaeute** [comment]
For now, I'm using this in the Debian packages:

```diff
From 4004503b7ce45da4ce1ff689d464075a7f017f45 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?IOhannes=20m=20zm=C3=B6lnig?= <zmoelnig@iem.at>
Date: Wed, 18 Jan 2023 09:19:14 +0100
Subject: [PATCH] Print renderer when invoking SSR via the generic wrapper

---
 data/ssr | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/data/ssr b/data/ssr
index 4b8c494..fbcc9ff 100755
--- a/data/ssr
+++ b/data/ssr
@@ -37,6 +37,9 @@ while [[ $1 ]]; do
   shift
 done
 
+echo "Using '${SSR_EXECUTABLE#ssr-}' SoundScape Renderer"
+echo ""
+
 SSR_EXECUTABLE=$(dirname $0)/$SSR_EXECUTABLE
 
 if [ ! -x $SSR_EXECUTABLE ]
-- 
2.39.0
```

But I think it would be nicer to add this information directly to the boilerplate.

**Comment by mgeier** [comment]
Thanks for the report!

I have created #324, what do you think about that?

The `ssr` script was only meant for backwards compatibility, I wanted to remove it at some point: #325.

**Comment by mgeier** [comment]
This was solved by #324 and #325.
